CONTRIBUTING.rst

```diff
@@ -0,0 +1,92 @@
+Contributing to CDAP
+====================
+
+Here are instructions to get you started if you want to contribute to CDAP. 
+
+Security Reports
+================
+
+Please *DO NOT* file a issue for security related problems. Please send your reports to [security@cask.co](mailto:security@cask.co).
+
+Creating Issues
+===============
+
+Inorder to file bugs or new feature requests please use [http://issues.cask.co](http://issues.cask.co) to file issues.
```

Comment:
typo: In order? Also maybe move the "to file issues." at the end to reduce some redundancy.